web_src/js/features/repo-issue.js

```diff
@@ -287,6 +287,20 @@ export function initRepoPullRequestMergeInstruction() {
   });
 }
 
+export function initRepoPullRequestAllowEditsFromMaintainers() {
+  $('#allow-edits-from-maintainers input').on('change', function () {
+    let url = '';
+    if ($(this).is(':checked')) { // allow edits
```

Comment:
```suggestion
    if (this.checked) { // allow edits
```

No need to go overboard with jQuery, we should reduce usage.